lib/matplotlib/animation.py

```diff
@@ -564,26 +562,25 @@ def finish(self):
             try:
                 stdout = [s.decode() for s in self._proc._stdout_buff]
                 stderr = [s.decode() for s in self._proc._stderr_buff]
-                verbose.report("MovieWriter.finish: stdout: %s" % stdout,
-                               level='helpful')
-                verbose.report("MovieWriter.finish: stderr: %s" % stderr,
-                               level='helpful')
+                log = logging.getLogger(__name__)
+                log.info("MovieWriter.finish: stdout: %s", stdout)
+                log.info("MovieWriter.finish: stderr: %s", stderr)
             except Exception as e:
                 pass
             msg = ('Error creating movie, return code: ' +
                    str(self._proc.returncode) +
-                   ' Try setting mpl.verbose.set_level("helpful")')
+                   ' Try setting python logging level to INFO')
```

Comment:
I think the logs about should just be moved to warning level?...

Reply:
`helpful` was equivalent to `INFO` so thats what I used.  level is already WARNING by deafult if logging is turned on.

Reply:
my point is that there isn't much of a point to have a warning-level message that says "this message is useless but you can replace it by a useful message by running with a different log-level setting" -- you may as well directly give the answer!

Reply:
I see now.  Yes, I made those `log.error` and they should preceed the RuntimeError message.